src/style/stylize.rs

```diff
@@ -179,8 +179,8 @@ impl<'a, T, U> Stylize<'a, T> for U
 where
     U: Styled<Item = T>,
```

Comment:
Ah… maybe this trait needs to be protected so it cant be implemented outside of ratatui… then it wouldnt be a breaking change to change this trait?

Reply:
I'm fine with the breaking change - it's pretty unlikely anyone has implemented Stylize manually on a widget (it's difficult to write a github search query to prove this however).

Reply:
Arguably this is a mistake in our current code and this PR fixes it. 

And if someone has implemented `Stylize` they would have noticed the inconsistency between `fg()` and `bg()` and (hopefully) reported it. That gives me a lot of confidence that this will not break anyone's code.

Reply:
yup - it could still be worth adding to the breaking changes doc, mainly because it's easier to just say all breaking changes are documented rather than all breaking changes that we think matter are.

Reply:
I've added a section for v0.26.1 into the `BREAKING-CHANGES.md` file.